packages/e2e-tests/specs/editor/various/change-detection.test.js

```diff
@@ -1,417 +0,0 @@
-
-		// Hold the posts request so we don't deal with race conditions of the
-		// save completing early.
-		await interceptSave();
-
-		// Keyboard shortcut Ctrl+S save.
-		await pressKeyWithModifier( 'primary', 'S' );
-
-		// Start this check immediately after save since dirtying the post will
-		// remove the "Saved" with the Save button.
-		const savedPromise = page.waitForSelector(
-			'.editor-post-saved-state.is-saved'
-		);
-
-		// Dirty post while save is in-flight.
-		await canvas().type( '.editor-post-title__input', '!' );
-
-		// Allow save to complete. Disabling interception flushes pending.
-		await Promise.all( [ savedPromise, releaseSaveIntercept() ] );
-
-		await assertIsDirty( true );
-	} );
-
-	it( 'Should prompt if block revision is made while save is in-flight, and save completes', async () => {
-		await canvas().type( '.editor-post-title__input', 'Hello World' );
-
-		// Hold the posts request so we don't deal with race conditions of the
-		// save completing early.
-		await interceptSave();
-
-		// Keyboard shortcut Ctrl+S save.
-		await pressKeyWithModifier( 'primary', 'S' );
-
-		// Start this check immediately after save since dirtying the post will
-		// remove the "Saved" with the Save button.
-		const savedPromise = page.waitForSelector(
-			'.editor-post-saved-state.is-saved'
-		);
-
-		await clickBlockAppender();
-		await page.keyboard.type( 'Paragraph' );
-
-		// Allow save to complete. Disabling interception flushes pending.
-		await Promise.all( [ savedPromise, releaseSaveIntercept() ] );
-
-		await assertIsDirty( true );
-	} );
-
-	it( 'should save posts without titles and persist and overwrite the auto draft title', async () => {
-		// Enter content.
-		await clickBlockAppender();
-		await page.keyboard.type( 'Paragraph' );
-
-		// Save.
-		await saveDraft();
-
-		// Verify that the title is empty.
-		const title = await canvas().$eval(
-			'.editor-post-title__input',
-			// Trim padding non-breaking space.
-			( element ) => element.textContent.trim()
-		);
-		expect( title ).toBe( '' );
-
-		// Verify that the post is not dirty.
-		await assertIsDirty( false );
-	} );
-
-	it( 'should not prompt to confirm unsaved changes when trashing an existing post', async () => {
-		// Enter title.
-		await canvas().type( '.editor-post-title__input', 'Hello World' );
-
-		// Save.
-		await saveDraft();
-		const postId = await page.evaluate( () =>
-			window.wp.data.select( 'core/editor' ).getCurrentPostId()
-		);
-
-		// Trash post.
-		await openDocumentSettingsSidebar();
-		await page.click( '.editor-post-trash.components-button' );
-		await page.click( '.components-confirm-dialog .is-primary' );
-
-		await Promise.all( [
-			// Wait for "Saved" to confirm save complete.
-			await page.waitForSelector( '.editor-post-saved-state.is-saved' ),
-
-			// Make sure redirection happens.
-			await page.waitForNavigation(),
-		] );
-
-		expect(
-			isCurrentURL(
-				'/wp-admin/edit.php',
-				`post_type=post&ids=${ postId }`
```

Comment:
This is actually wrong. `isCurrentURL` only returns false when the url is not a WP url, it doesn't check the query params for some reason. The final url also doesn't include the `ids=${postId}` part either.